pkgs/build-support/docker/default.nix

```diff
@@ -539,6 +566,8 @@ rec {
       buildVMMemorySize ? 512
     , # Time of creation of the image.
       created ? "1970-01-01T00:00:01Z"
+    , # Compressor to use. One of: none, gz, zstd.
```

Comment:
Needs documentation in the manual, `doc/build-helpers/images/dockertools.section.md`